mycore-restapi/src/main/java/org/mycore/restapi/annotations/MCRParam.java

```diff
@@ -19,7 +19,10 @@
 package org.mycore.restapi.annotations;
 
 import java.lang.annotation.Annotation;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
 
+@Retention(RetentionPolicy.RUNTIME)
```

Comment:
MCRParam should be generated at runtime and not be used to annotate static values

Reply:
Without retention policy the params where not found in code, even if created dynamically